utilities/xmlrpcpp/test/CMakeLists.txt

```diff
@@ -3,6 +3,23 @@ if(TARGET xmlrpcvalue_base64)
   target_link_libraries(xmlrpcvalue_base64 xmlrpcpp)
 endif()
 
+# Some of the tests that follow use boost threads.
```

Comment:
Please add `<test_depend>boost</test_depend>`, since inclusion of this file is guarded by `CATKIN_ENABLE_TESTING`.

Reply:
Done.